src/mongocxx/exception/error_category.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 MongoDB Inc.
+// Copyright 2015 MongoDB Inc.
```

Comment:
FYI - Generally, we don't update the year in copyright headers. No need to change it.